native/src/hooks/useTtsPlayer.ts

```diff
@@ -17,8 +16,7 @@ const useTtsPlayer = (model?: PageModel | LocalNewsModel | TunewsModel | undefin
   useFocusEffect(
     useCallback(() => {
       if (model && model.content.length > 0) {
-        const content = parseHTML(model.content)
-        const sentences: string[] = segment(languageCode, content)
+        const sentences: string[] = segmentation(languageCode, addingPeriodsToDom(model.content))
```

Comment:
:question: Why do we need periods here now?

Reply:
hmm .. not actually needed for native but keeping the web and native consistent.
In general I think tts for native works differently and as I remember it didn't need a period to figure out when to stop unlike web.

Reply:
Could reproduce on firefox linux.

Reply:
> Actually, I can't. Everything works as expected for me and the tts correctly jumps to the next sentence after each utterance/sentence, even without adding extra periods. Could you please test this and add concrete info on how to reproduce this?

Comment out the following lines at `ttsDomManipulation.ts`:
  appendPeriod(paragraphs)
  appendPeriod(listItems)

At firefox (anything doesn't have a full stop at the end will continue reading like lists): 
http://localhost:9000/muenchen/en/everyday-life/information-on-insurance-contracts-shopping-invoices-reminders-finances/bank-account